src/oadp_resources/oadp/data_protection_application.py

```diff
@@ -1,6 +1,28 @@
 from oadp_constants.resources import ApiGroups
 from ocp_resources.resource import NamespacedResource
+from enum import Enum
 
 
 class DataProtectionApplication(NamespacedResource):
     api_group = ApiGroups.OADP_API_GROUP.value
+
+    class Condition(Enum):
+        class Status(Enum):
+            TRUE = "True"
+            FALSE = "False"
+
+        class Type(Enum):
+            RECONCILED = "Reconciled"
+
+    def is_reconciled(self):
```

Comment:
can you change to reconciled instead of is_

Reply:
Sure, I just thought it will be more clear this way.